cmd/kpod/save.go

```diff
@@ -91,6 +91,9 @@ func saveCmd(c *cli.Context) error {
 	for _, image := range args {
 		dest := dst + ":" + image
 		if err := runtime.PushImage(image, dest, saveOpts); err != nil {
+			if err2 := os.Remove(output); err2 != nil {
```

Comment:
I think you would just logrus this.  But return the original error.

Reply:
fixed